portafly/src/components/data-list/SearchWidget.tsx

```diff
@@ -196,10 +194,10 @@ const SearchWidget: React.FunctionComponent<ISearchWidget> = ({ categories }) =>
   ), [selectedCategory])
 
   return (
-    <DataToolbarGroup variant="filter-group">
-      { categorySelect }
-      { appliedFilters }
-      { selectedCategory.options
+    <ToolbarGroup variant="filter-group">
+      {categorySelect}
+      {appliedFilters}
+      {selectedCategory.options
```

Comment:
These unnecessary changes were triggered by tslint? if not, avoid them

Reply:
done!